app/src/main/java/org/fossasia/openevent/general/event/Event.kt

```diff
@@ -78,5 +89,58 @@ data class Event(
     @ColumnInfo(index = true)
     @Relationship("event-sub-topic", resolve = true)
     var eventSubTopic: EventSubTopic? = null
-
 )
+
+object DataBindingAdapter {
+
+    /**
+     * Toggle Visibillity of the view depending upn the value is null or not.
+     */
+    @BindingAdapter("android:visibility")
+    @JvmStatic
+    fun setVisibility(view: View, value: String?) {
```

Comment:
boolean

Reply:
@iamareebjamal i'm checking whether the value of text is null or not why would i'll be using boolean

Reply:
How and why will it be used?

Reply:
It works like if text value is null then this view gets invisible and if it is not then view gets visible

Reply:
@iamareebjamal it is quite a useful function and reduces a lot of code probably the best thing about data binding so far

Reply:
Then rename to `hideIfEmpty`

Reply:
It would be convenient to remap the property otherwise one may use both of the properties

Reply:
I don't understand?

Reply:
They should be able to use both properties

Reply:
@iamareebjamal updated

Reply:
@iamareebjamal if it's okay should I continue refactoring entire project in on go or make separate PR's